dev/KairaGupta/importance_score/importance_score.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from sklearn import metrics
+import matplotlib.pyplot as plt
+
+def imp_score_singular_datapoints(model, train_features, train_labels, test_features, test_labels, i_start, i_end):
+    """
+        Inputs: 
+            model: model object which must have function model.fit(train_features, train_labels)
+            train_features: numpy array of feature values for training
+            train_labels: numpy array of correct labels for train_features
+            test_features: numpy array of feature values for test data
+            test_labels: numpy array of correct labels for test data
+            i_start: start index to remove features from train_labels (one-by-one)
+            i_end: end index (not inclusive) to remove features from train_labels (one-by-one)
+        Output:
+            Plot of difference in performance scores (accuracy and f-score) on the test set by removing data points (one-by-one) from training data.
+    """
+    
+    fs_list = []
+    acc_list = []
+    
+    temp_model = model
+    temp_model.fit(train_features, train_labels)
+    
+    pred_full_data = temp_model.predict(test_features)
+
+    fs_full_data = metrics.f1_score(test_labels, pred_full_data)
+    acc_full_data = metrics.accuracy_score(test_labels, pred_full_data)
+    
+    for i in range(i_start,i_end):
+        temp_train = np.concatenate((train_features[:i], train_features[i+1:]))
+        temp_labels = np.concatenate((train_labels[:i], train_labels[i+1:]))
+        temp_model = model
+        temp_model.fit(temp_train, temp_labels)
```

Comment:
nit: this line could be directly called passing `np.concatenate((train_features[:i], train_features[i+1:]` and `np.concatenate((train_labels[:i], train_labels[i+1:]))` and still be very readable.
I understand that assigning to a new variable on each iteration looks cleaner, but there is a fine line between readability and unnecessary operations.

Again, this PR is solid and I dot really have any complaints, these are just styling preferences.

Reply:
I'll keep this in mind in my future PRs as well. Thanks!

Reply:
@mlopatka If I wish to keep it as it for now (I would like to work on other issues) and make changes in future, how do I merge this (as you have told in above comment)?

Reply:
oh! sorry for the delayed reply. This PR was already approved.
I'll merge it in now. 
In the future, if a PR is approved you can also feel free to merge it into master.